src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java

```diff
@@ -674,22 +608,14 @@ public static double log10SumLog10(final double[] log10Values, final int start,
 
         // we may decide to just normalize in log space without converting to linear space
         if (keepInLogSpace) {
-            for (int i = 0; i < array.length; i++) {
-                array[i] -= maxValue;
-            }
-            return array;
+            return Arrays.stream(array).map(x -> x - maxValue).toArray();
```

Comment:
i worry a bit about the performance of this stream operations here. Arrays are fast but streams create a lot of garbage and these math operations are heavily hit during some operations